### PR TITLE
reference maintainer upgrade guide

### PR DESCRIPTION
**Pull request by dataders**
## What are you changing in this pull request and why?

add link to 1.5 adapter maintainer upgrade guide GitHub discussion (https://github.com/dbt-labs/dbt-core/discussions/7213) to the generic 1.5 upgrade guide

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/guides/migration/versions/02-upgrading-to-v1.5.md [hunk 1]
```diff
@@ -59,3 +59,10 @@ More to come!
 
 ### dbt-core Python API
 - Auto-generated documentation ([#2674](https://github.com/dbt-labs/docs.getdbt.com/issues/2674)) for dbt-core CLI & Python API for programmatic invocations
+
+
+### Materialized View, Materialized Tests
```
Comment (mirnawong1) [inline website/docs/guides/migration/versions/02-upgrading-to-v1.5.md]: sugg sentence case and shortening it

```suggestion
### Materialized views and tests
```

**Comment by mirnawong1** [comment]
small sugg @dataders - let me know what you think and happy to approve this for you!

File: website/docs/guides/migration/versions/02-upgrading-to-v1.5.md [hunk 2]
```diff
@@ -63,3 +63,10 @@ More to come!
 
 ### dbt-core Python API
 - Auto-generated documentation ([#2674](https://github.com/dbt-labs/docs.getdbt.com/issues/2674)) for dbt-core CLI & Python API for programmatic invocations
+
+
+### Materialized views and tests
+
+:::caution Under construction 🚧
```
Comment (mirnawong1) [inline website/docs/guides/migration/versions/02-upgrading-to-v1.5.md:70]: @dataders actually is this suppose to have a link to anything? (eg. https://github.com/dbt-labs/dbt-core/discussions/7213)?

Comment (mirnawong1) [reply website/docs/guides/migration/versions/02-upgrading-to-v1.5.md:70]: Hey @dataders wanted to check in with this?

**Comment by mirnawong1** [comment]
hey @dataders - checking in on this pr?

**Comment by matthewshaver** [comment]
Closing as the link has since been added to the [v1.5 upgrade guide](https://docs.getdbt.com/guides/migration/versions/upgrading-to-v1.5#for-maintainers-of-adapter-plugins)

**Pull request closed without merging**